haskell_scotty/Server.hs

```diff
@@ -18,8 +18,8 @@ main = scotty 6969 $ do
 
   post "/api/clients" $ do
     client <- jsonData
-    client <- liftIO $ saveClient client
-    jsonAPI client
+    client' <- liftIO $ saveClient client
```

Comment:
Yeah. You can prime any variable; it's just another variable-name-character. But this is the way it's normally used.